pvlib/pvsystem.py

```diff
@@ -2037,6 +2051,120 @@ def snlinverter(v_dc, p_dc, inverter):
     return ac_power
 
 
+def adrinverter(v_dc, p_dc, inverter, vtol=0.10):
+    r'''
+    Converts DC power and voltage to AC power using Anton Driesse's
+    Grid-Connected PV Inverter efficiency model
+
+    Parameters
+    ----------
+    v_dc : numeric
+        A scalar or pandas series of DC voltages, in volts, which are provided
+        as input to the inverter. If Vdc and Pdc are vectors, they must be
+        of the same size. v_dc must be >= 0. (V)
+
+    p_dc : numeric
+        A scalar or pandas series of DC powers, in watts, which are provided
+        as input to the inverter. If Vdc and Pdc are vectors, they must be
+        of the same size. p_dc must be >= 0. (W)
+
+    inverter : dict-like
+        A dict-like object defining the inverter to be used, giving the
+        inverter performance parameters according to the model
+        developed by Anton Driesse [1].
+        A set of inverter performance parameters may be loaded from the
+        supplied data table using retrievesam.
+        See Notes for required keys.
+
+    vtol : numeric
+        A unit-less fraction that determines how far the efficiency model is allowed
+        to extrapolate beyond the inverter's normal input voltage operating range.
+        0.0 <= vtol <= 1.0
+
+    Returns
+    -------
+    ac_power : numeric
+        A numpy array or pandas series of modeled AC power output given the
+        input DC voltage, v_dc, and input DC power, p_dc. When ac_power would
+        be greater than pac_max, it is set to p_max to represent inverter
+        "clipping". When ac_power would be less than -p_nt (energy consumed
+        rather  than produced) then ac_power is set to -p_nt to represent
+        nightly power losses. ac_power is not adjusted for maximum power point
+        tracking (MPPT) voltage windows or maximum current limits of the
+        inverter.
+
+    Notes
+    -----
+
+    Required inverter keys are:
+
+    ======   ============================================================
+    Column   Description
+    ======   ============================================================
+    p_nom    The nominal power value used to normalize all power values,
+             typically the DC power needed to produce maximum AC power output,
+             (W).
+
+    v_nom    The nominal DC voltage value used to normalize DC voltage values,
+             typically the level at which the highest efficiency is achieved,
+             (V).
+
+    pac_max  The maximum AC output power value, used to clip the output if
```

Comment:
A longer Column header by one = sign. You'll also need to add one space between all of the Column and the Description sections so that the table stays properly formatted. I guess you should probably also make the Description header longer and/or make the text fit within the existing Description header. 

I'm pretty sure that Sphinx would build the table in its present form and spit out a warning. It's not a big deal if it's just one table, but every silly warning makes it slightly harder to identify other issues with the documentation. If you actually build the docs and don't see a warning then we can ignore all this.